backend/store/postgres.go

```diff
@@ -300,3 +314,39 @@ func (s *CollaborationStoreImpl) FindGroupAdmissionByID(admissionID string) (gro
 func (s *CollaborationStoreImpl) DeleteGroupAdmission(admission *model.GroupAdmission) error {
 	return s.db.Delete(admission).Error
 }
+
+func (s *SrsStoreImpl) CreateRevlog(newRev *model.Revlog) error {
+	return s.db.Create(&newRev).Error
+}
+
+func (s *SrsStoreImpl) GetCardBinding(userID string, cardID string) (userCardBinding *model.UserCardBinding, err error) {
+	if err = s.db.Where(model.UserCardBinding{UserID: userID, CardID: cardID}).First(&userCardBinding).Error; errors.Is(err, gorm.ErrRecordNotFound) {
+		err = helper.ErrStoreNoEntryWithID
+	}
+	return
+}
+
+func (s *SrsStoreImpl) GetDeckCards(userID string, deckID string) (userCards []*model.UserCardBinding, err error) {
+	if err = s.db.Where(model.UserCardBinding{DeckID: deckID}).Find(&userCards).Error; errors.Is(err, gorm.ErrRecordNotFound) {
+		err = helper.ErrStoreNoEntryWithID
+	}
+	return
+}
+
+func (s *SrsStoreImpl) CreateUserCard(newCard *model.UserCardBinding) error {
+	return s.db.Create(newCard).Error
+}
+
+func (s *SrsStoreImpl) ModifyUserCard(userCard *model.UserCardBinding) (err error) {
+	err = s.db.Save(&model.UserCardBinding{
+		ID:      userCard.ID,
+		UserID:  userCard.UserID,
+		CardID:  userCard.CardID,
+		DeckID:  userCard.DeckID,
+		Type:    userCard.Type,
+		Due:     userCard.Due,
+		LastIvl: userCard.LastIvl,
+		Factor:  userCard.Factor,
+	}).Error
+	return
+}
```

Comment:
```suggestion
func (s *SrsStoreImpl) ModifyUserCard(userCard *model.UserCardBinding) error {
	return s.db.Save(&model.UserCardBinding{
		ID:      userCard.ID,
		UserID:  userCard.UserID,
		CardID:  userCard.CardID,
		DeckID:  userCard.DeckID,
		Type:    userCard.Type,
		Due:     userCard.Due,
		LastIvl: userCard.LastIvl,
		Factor:  userCard.Factor,
	}).Error
}
```
Maybe just return the error

Reply:
👍🏼⚡ 75f4f57